kanidmd_web_ui/pkg/README.md

```diff
@@ -0,0 +1,101 @@
+
```

Comment:
Similar, is there a reason to duplicate this?

Reply:
see above, fixing whinge from wasm

Reply:
It can't use relative paths?

Reply:
Not as far as this (and some other searching) would indicate... https://github.com/rustwasm/wasm-pack/issues/718

Reply:
Cool cool.